docs/menu.md

```diff
@@ -55,7 +55,7 @@ export class AppComponent implements OnInit, OnDestroy {
 
 ## Custom Menu
 
-The editorView will be available from the init function
+The editorView will be available from the init function. Create `customMenu` as a template variable using the custom-menu component.
```

Comment:
```suggestion
Create `customMenu` as a template variable using the custom-menu component.
```

init function no longer exists. can you remove this line

Reply:
> Create `customMenu` as a template variable using the custom-menu component.

this somehow feels like the editor is providing a custom menu component to create custom, but it is create a template and pass the ref as a input to the menu component, via `customMenuRef`